overview/encryption.md

```diff
@@ -6,35 +6,32 @@ permalink: /overview/encryption/
 * TOC
 {:toc}
 
-Many of the [KBC components](/overview/) use the Encryption API; it encrypts sensitive values
-which are supposed to be securely stored and decrypted inside the component itself.
+Many [KBC components](/overview/) use the Encryption API to encrypt sensitive values
+intended for secure storage. These values are then decrypted within the component itself. 
+This process ensures that the encrypted values are only accessible inside the components and not
+by API users. Additionally, no decryption API is available, meaning end-users cannot decrypt
+these values.
 
-This means that the encrypted values are available inside the components and are not accessible
-to the API users. Also, there is no decryption API and there is no way the end-user can decrypt
-the encrypted values.
-
-Decryption is only executed when serializing configuration to the configuration file for
-the Docker container. The decrypted data are stored on the Docker host drive and are
-deleted immediately after the container finishes. The actual component code always reads
-the decrypted data.
+Decryption occurs solely during the serialization of configuration to the Docker container's 
+configuration file. The decrypted data are stored on the Docker host drive and are promptly 
+deleted after the container's completion. The component code exclusively accesses the decrypted data.
 
 ## UI Interaction
-When saving arbitrary configuration data, values marked by the `#` character are automatically encrypted.
-Given the following configuration:
+When saving arbitrary configuration data, values prefixed with the `#` character are automatically encrypted.
```

Comment:
No to to není úplně dobře - v konfiguraci (kterou uživatel uvidí třeba v raw modu nebo na API) je to v jsonu a ten prefix se týká klíče a ne hodnoty - in short chceme říct, že `"#foo": "bar"` se zašifruje a ne, že se zašifruje `"foo": "#bar"`